src/main/java/easy/mind/com/api/Controller/UserController.java

```diff
@@ -0,0 +1,51 @@
+package easy.mind.com.api.Controller;
+
+import easy.mind.com.api.DTO.UserDTO;
+import easy.mind.com.api.Entity.User;
+import easy.mind.com.api.Srvice.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/users")
```

Comment:
Advice: I personally prefer to sort annotations by type or by length of annotation. That allows easier scanning of annotations.

By type: for example, on DB Entity classes - 1st group is persistent annotations, 2nd group is Lombok.
By length: simply rearrange annotations by it's name length. 
```
@RestController
@RequestMapping("/users")
@RequiredArgsConstructor
```